ska_service/setup.cfg

```diff
@@ -0,0 +1,33 @@
+[metadata]
+name = ska_service
+version = attr: ska_service.__version__
+author = Markus Johansson
+author_email = markus.h.johansson@skane.se
```

Comment:
Do you want your email in a public file in a public-facing repo? Guess crawlers will find it and put it into "open-to-spam" collections of emails.